### DISCUSSION TITLE
Locked Function Error

### DISCUSSION THREAD

**Discussion by Scaver** [SFX-WoW/Masque#166] [Issues] [answered]
1x [ADDON_ACTION_BLOCKED] AddOn 'Masque' hat versucht die geschützte Funktion '<unnamed>:Hide()' aufzurufen.
!BugGrabber\BugGrabber.lua:573: in function <!BugGrabber\BugGrabber.lua:573>
[C]: in function `Hide'
FrameXML\WorldMapActionButton.lua:58: in function`Clear'
FrameXML\WorldMapActionButton.lua:67: in function `Refresh'
FrameXML\WorldMapActionButton.lua:19: in function`SetMapAreaID'
FrameXML\WorldMapFrame.lua:390: in function <FrameXML\WorldMapFrame.lua:353>
[C]: in function `Hide'
FrameXML\UIParent.lua:2320: in function`SetUIPanel'
FrameXML\UIParent.lua:2347: in function `MoveUIPanel'
FrameXML\UIParent.lua:2378: in function`HideUIPanel'
FrameXML\UIParent.lua:2052: in function <FrameXML\UIParent.lua:2044>
[C]: in function `SetAttribute'
FrameXML\UIParent.lua:2846: in function <FrameXML\UIParent.lua:2833>
[C]: in function`HideUIPanel'
FrameXML\UIParent.lua:2812: in function `ToggleFrame'
FrameXML\WorldMapFrame.lua:219: in function`ToggleWorldMap'
[string "TOGGLEWORLDMAP"]:1: in function <[string "TOGGLEWORLDMAP"]:1>

Locals:

**Comment by StormFX** [comment, marked as answer]
This is not an issue with Masque.
